.github/workflows/post_merge_integration.yml

```diff
@@ -76,11 +76,11 @@ jobs:
 
   test-api:
     name: Python - Test with coverage
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     strategy:
       matrix:
         # Match versions for python + R to Ubuntu 22.04 LTS at the time of writing.
```

Comment:
nit: Update comment to 'Ubuntu 24.04'.

Reply:
Done in: https://github.com/bcgov/wps/pull/3995/commits/095b46bf148446caf78f0c9c0eca7bb747d0e5c7